### PR TITLE
Enter charger id (mockup)

### LINKED ISSUE #20: Enter charger ID (mockup)
- [x] Make an input field for the user to enter charger ID (Charger ID is six digits)

- [x] Verify if the charger ID is valid or not
-  [x] Show a connection message to the user if the charger ID is valid
-  [x] Show an error message to the user if the charger ID is invalid

input_screen
input_screen_numpad
input_screen_not_identified

ESTIMATION: 5

### PR DESCRIPTION
**Pull request by filipfloden**
Fixes #20 

# Description

This PR is of the enter charger id page with mock chargers to test functionality.

## Lists of changes

- [x] Created the map as a base
- [x] Charge button which opens the 'enter charger id' page
- [x] Enter charger id page with text field to enter an id of 6 integer
- [x] Status messages for entered id

## Testing

- [ ] Make sure the button works as intended (open the charger page)
- [ ] Dragging the charger page to dismiss or click outside page to dismiss
- [ ] Make sure only numbers can be entered in as the id

### CODE REVIEW

**Review by LucasStrand — APPROVED** [review]
I got no complaints, good work people 🎊

**Review by Yevve — APPROVED** [review]
Good job , very good mockup version👌

File: FlexiCharge/FlexiCharge/Views/IdentifyCharger/IdentifyChargerView.swift [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+            isButtonVisible = 1
+            buttonTextColor = Color(red: 0.30, green: 0.30, blue: 0.30)
+        }
+    }
+}
+
+func startCharging(){
+    //Add functionality to startChargingButton
+    //Send all selected options to API
+}
```
Comment (scamdivine) [inline FlexiCharge/FlexiCharge/Views/IdentifyCharger/IdentifyChargerView.swift:138]: this seems like a silly mistake 
cleaner code 
`func startCharging() {`

File: FlexiCharge/FlexiCharge/Views/Content/ContentView.swift [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+struct ContentView: View {
+    let screenHeight = UIScreen.main.bounds.size.height
+    let screenWidth = UIScreen.main.bounds.size.width
+    @State var offset: CGFloat = 0
+    @State var lastOffset: CGFloat = 0
+    @State var keyboardHeight: CGFloat = 0
+    @GestureState private var gestureOffset: CGFloat = 0
+    
+    var body: some View {
+        ZStack (alignment: .bottom){
```
Comment (scamdivine) [inline FlexiCharge/FlexiCharge/Views/Content/ContentView.swift:21]: another blank line between declarations and code

File: FlexiCharge/FlexiCharge/Views/IdentifyCharger/IdentifyChargerView.swift [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    @State private var isButtonDisabled: Bool = true
+    @State private var isButtonVisible: Double = 0
+    @State private var buttonTextColor: Color = .clear
+    @State var value: CGFloat = 0
+    @State var keyboardHeight: CGFloat = 0
+
+
+    var body: some View {
+        ZStack (alignment: .top){
+            RoundedRectangle(cornerRadius: 5)
```
Comment (scamdivine) [inline FlexiCharge/FlexiCharge/Views/IdentifyCharger/IdentifyChargerView.swift:27]: cleaner code `ZStack (alignment: .top) {`

**Review by scamdivine — CHANGES_REQUESTED** [review]

File: FlexiCharge/FlexiCharge/Views/IdentifyCharger/IdentifyChargerView.swift (hunk 1 above)
Comment (filipfloden) [reply FlexiCharge/FlexiCharge/Views/IdentifyCharger/IdentifyChargerView.swift:138]: hm I wonder who made this line of code 🤔

File: FlexiCharge/FlexiCharge/Views/Content/ContentView.swift (hunk 2 above)
Comment (filipfloden) [reply FlexiCharge/FlexiCharge/Views/Content/ContentView.swift:21]: I personally think this line should be there too seperate declared variables and the start of a view

File: FlexiCharge/FlexiCharge/Views/IdentifyCharger/IdentifyChargerView.swift (hunk 1 above)
Comment (scamdivine) [reply FlexiCharge/FlexiCharge/Views/IdentifyCharger/IdentifyChargerView.swift]: let us agree to blame this on @LucasStrand

Comment (LucasStrand) [reply FlexiCharge/FlexiCharge/Views/IdentifyCharger/IdentifyChargerView.swift]: 😳

**Pull request merged**